assets/acquia_lift.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file
+ * Contains the definition of the behavior openAcquiaLiftExperienceBuilder.
+ */
+
+(function ($, Drupal) {
+
+  'use strict';
+
+  /**
+   * Opens the Acquia Lift Experience Builder, if it is available.
+   */
+  function openExperienceBuilder() {
+    if (window.AcquiaLiftPublicApi) {
+      window.AcquiaLiftPublicApi.activate();
+    }
+    else {
+      alert('Sorry, Acquia Lift could not be found');
+    }
```

Comment:
I don't have a hard opinion. The text came from the link I already had. But sure, we can change it.